backend/app/controllers/enrollments_controller.rb

```diff
@@ -224,6 +224,26 @@ def next_enrollments
       root: "enrollments"
   end
 
+  # GET enrollment/1/mark_demandeur_notify_events_as_processed/1
+  def mark_demandeur_notify_events_as_processed
+    @enrollment = policy_scope(Enrollment).where(id: params[:id])
```

Comment:
```suggestion
    @enrollment = policy_scope(Enrollment).find(id: params[:id])
```

Un `where` renvoi systématiquement un tableau et non un modèle unique